lib/broccoli/ember-app.js

```diff
@@ -1448,9 +1448,9 @@ class EmberApp {
   */
   styles() {
     if (!this._cachedStylesTree) {
-      if (existsSync(`${this._stylesPath}/${this.name}.css`)) {
-        throw new SilentError(`Style file cannot have the name of the application - ${this.name}`);
-      }
+      //if (existsSync(`${this._stylesPath}/${this.name}.css`)) {
```

Comment:
If we are going to remove it, let’s remove it. Commenting seems bad....

Reply:
ofc, Good Sir. I opened this PR just to start a conversation.